server.js

```diff
@@ -50,6 +51,16 @@ app.get('/weather', (request, response) => {
   }
 });
 
+////////route for events/////////
+// app.get('/events', (request, response) => {
+//     try {
+
+//         const key = process.env.EVENTFUL_API_KEY;
+//         const url = `http://api.eventful.com/rest/events/search?...&where=$%7Blocation.latitude%7D,$%7Blocation.longitude%7D&within=5%60`
+//     }
```

Comment:
http://api.eventful.com/json/events/search?location=${location}&date=Future&app_key=${process.env.EVENTFUL_API_KEY}

Reply:
The ... is just to show you can put more queries in the url